Sources/Classes/CKRecord+NSManagedObject.swift

```diff
@@ -75,10 +75,10 @@ extension CKRecord {
             let referencesValuesDictionary = self.dictionaryWithValues(forKeys: self.allReferencesKeys(usingRelationshipsByNameFromEntity: entity.relationshipsByName))
             var managedObjectsDictionary: Dictionary<String,AnyObject> = Dictionary<String,AnyObject>()
             for (key,value) in referencesValuesDictionary {
-               /* if (value as? String) != nil && (value as! String) == SMStore.SMCloudRecordNilValue {
-                    managedObjectsDictionary[key] = SMStore.SMCloudRecordNilValue as AnyObject?
-                    continue
-                }*/
+                /* if (value as? String) != nil && (value as! String) == SMStore.SMCloudRecordNilValue {
```

Comment:
Let's just delete this obsolete code instead of leaving it around as a comment; I can do it after the PR merge if you like